doc/ref_model/chapters/chapter01.md

```diff
@@ -11,7 +11,7 @@
 
 <a name="1.1"></a>
 ## 1.1 Overview
-The Reference Model (RM) specification is written on the basis of complete infrastructure abstraction and exposing set of capabilities, resources, and interfaces to workloads which they will be written against. The aim of Reference Model is to be agnostic across VM-based and container-based workloads and this document should act as a "catalogue" for VNF/CNF application designers to understand everything they need to know from the underlying infrastructure they are writing their application to run on.
+The Reference Model (RM) specification is written on the basis of complete infrastructure abstraction and exposing set of capabilities, resources, and interfaces to workloads which they will be written against. The aim of Reference Model is to be agnostic across VM-based and container-based workloads and this document should act as a "catalogue" for application designers to understand everything they need to know from the underlying infrastructure they are writing their application to run on.
```

Comment:
Last sentence appears to be incomplete -- mentions "... from the underlying ..." but no "to ...". Can we take this opportunity to makse some language corrections too?

May I suggest a rewrite. For example, as:

The Reference Model (RM) specifies complete infrastructure abstraction and the exposing of a set of capabilities, resources, and interfaces to workloads. The aim of the Reference Model is to be virtualisation technology agnostic (VM-based and container-based) and act as a "catalogue" of the exposed infrastructure capabilities, resources, and interfaces needed to develop the workloads.

Reply:
Thanks, done:)

Reply:
@EditKoselak Can you delete "complete" -- this was discussed the RM meeting of April 29, 2020. Thanks

Reply:
> @EditKoselak Can you delete "complete" -- this was discussed the RM meeting of April 29, 2020.
> 
> Also, from 1.4: Infrastructure profiles catalogue
> 
> Thanks

Thanks, done:)

Reply:
@pgoyal01 can you re-review, please?